app/controllers/clients_controller.rb

```diff
@@ -18,6 +18,15 @@ def index
     end
   end
 
+  def search
+    authorize Client
+    @q = policy_scope(Client).ransack contact_full_name_cont: params[:term]
```

Comment:
You could also do something like `contact_first_name_or_contact_last_name_cont` to avoid having to cache the `full_name` value in the DB.

Ransack also supports [defining aliases](https://github.com/activerecord-hackery/ransack#ransack-aliases) as a shortcut.

Reply:
I guess the downside is it won't support searching for the full name directly, but not sure if the users will expect to be able to do that (they will have to include the comma too)